plugins/woocommerce/src/Blocks/BlockTypes/MiniCart.php

```diff
@@ -483,9 +483,10 @@ protected function render_experimental_iapi_mini_cart( $attributes, $content, $b
 			wp_interactivity_state(
 				$this->get_full_block_name(),
 				array(
-					'totalItemsInCart'  => $cart_item_count,
-					'badgeIsVisible'    => $badge_is_visible,
-					'formattedSubtotal' => $formatted_subtotal,
+					'totalItemsInCart'     => $cart_item_count,
+					'badgeIsVisible'       => $badge_is_visible,
+					'formattedSubtotal'    => $formatted_subtotal,
+					'drawerOverlayClass'   => 'wc-block-components-drawer__screen-overlay wc-block-components-drawer__screen-overlay--with-slide-out wc-block-components-drawer__screen-overlay--is-hidden',
```

Comment:
Taking a look at the [derived state of this property](https://github.com/woocommerce/woocommerce/blob/17387ec18dce96e4ab506cbac11075c2834b565f/plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts#L108-L117), it looks like not all classes are dynamic.

Out of curiosity, did we consider having something like this and remove that prop:

```php
data-wp-class--wc-block-components-drawer__screen-overlay--with-slide-in="context.isOpen"
data-wp-class--wc-block-components-drawer__screen-overlay--is-hidden="!context.isOpen"
```